CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Contributing to Headset
+
+Welcome to the Headset community!
+
+## Community
+
+We have two main channels of communication:
+- our Slack workspace: https://tinyurl.com/y7m8y5x4
+- issues on GitHub
+
+If you have any questions on where to start and/or how to proceed, let us know!
+
+## Ways to contribute
+
+We welcome many ways of contributing to Headset:
+- finding bugs and [reporting issues](https://github.com/headsetapp/headset-electron/issues)
+- picking and working on an existing [issue](https://github.com/headsetapp/headset-electron/labels/help-wanted) marked as 
+  "help wanted"
+- proposing and implementing improvements to Headset
+
+## Setting up the environment
+
+1. Fork the repository on GitHub
+
+2. Clone your fork of the repository
+
+  ```
+  git clone https://github.com/<your-username>/headset-electron.git
+  ```
+
+3. Installation is OS-dependent, so choose the appropriate OS directory:
+  - `windows` for Windows
+  - `darwin` for OSX
+  - `linux` for Linux
+
+    To install all the required dependencies run:
+
+    ```
+    npm i && cd <your-os> && npm i
+    ```
+
+    which will install both global and OS-dependent packages.
+
+    Linux users also need to install appropriate dependencies for [dbus](https://www.npmjs.com/package/dbus#general)
+
+
+4. Now you can run the app:
+
+  ```
+  npm run production-like
+  ```
+
+You are ready to pick a "help wanted" issue and start working on it.
+
+## Testing
+
+Whenever you work on a new feature, be sure to add appropriate tests. Before 
+submitting your changes, you should run all the test suites to make sure it works
+as expected.
+
+1. Linter tests
+
+    To run linter tests use
+    ``` 
+    npm run lint <directory>
+    ```
+    where `<directory>` is either the OS you are working on (`darwin`, `linux`, 
+    `windows`), the test folder, or the shared folder. If you work on multiple OSs (or 
+    directories), you can run the same `npm run lint` followed by a list of all 
+    directories separated by spaces e.g. 
+    ```
+    npm run lint linux windows shared
+    ```
+
+2. General test suites
+
+    Run 
+    ```
+    npm run test:<os>
+    ```
+    where `<os>` is either `darwin`, `linux` or `windows`. If you're working on 
```

Comment:
Maybe mention that the tests are OS specific, so you do need to be running the test on the same OS or it will fail. You can still develop for multiple OS.
We usually use virtual machines (VirtualBox in my case) to do some tests if needed. But if people don't have access to other OSs, they should configure Travis and/or Appveyor on their forks so each time they push a commit, the CI will run the tests.
I don't think that mentioning the CI is necessary, but definitely, the tests are OS specific.